### PR TITLE
RStudio update to 2023.09 version

### PR DESCRIPTION
**Pull request by ogakulov**

### CODE REVIEW

File: rstudio/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# ###########################################################################
-#
+###########################################################################
+#  
 #  CLOUDERA COMMUNITY RUNTIMES
 #
-# (C) Cloudera, Inc. 2022
+# (C) Cloudera, Inc. 2021
```
Comment (peterableda) [inline rstudio/Dockerfile:5]: 2023

File: rstudio/Dockerfile [hunk 2]
```diff
@@ -39,78 +39,124 @@
...
-
 # Create a Symlink to the default editor launcher
 # This is the main requirement to get CML to launch a different editor. Create a symlink
 # from the editors launcher to /usr/local/bin/ml-runtime-editor
 RUN ln -s /usr/local/bin/rstudio-cml /usr/local/bin/ml-runtime-editor
+RUN export PATH=$PATH:/home/cdsw/R/x86_64-pc-linux-gnu-library/4.1
+USER root
+# MLX-50 Allow the engine image to be run as user cdsw upon startup.
+# To do this we must 'chmod 777' /etc related directories.
+RUN for i in /etc /var /etc/alternatives /var/lib/dpkg/ /home /home/dataiku /runtime-addons; do \
```
Comment (peterableda) [inline rstudio/Dockerfile:129]: I would assume that these are done in the base image, do we need to rerun that? 

Why is the dataiku folder?

File: rstudio/Dockerfile [hunk 3]
```diff
@@ -39,78 +39,124 @@
...
+ if [ -d ${i} ]; then chmod 777 ${i}; fi; \
+ done
+# DSE-11162 Base engine changes to support runtime addons.
+# To do this, we must 'chown cdsw' some directories.
+# Make root directory read/writable by cdsw user
+RUN chown cdsw /
+RUN chown cdsw /
```
Comment (peterableda) [inline rstudio/Dockerfile:136]: Do you need to rerun? Twice? :)

File: rstudio/Dockerfile [hunk 4]
```diff
@@ -39,78 +39,124 @@
...
+# Add permissions to cdsw user recursively to the common directories that gets touched as part
+# of the engine startup process.
+RUN for i in /bin /var /etc /opt /var/lib/dpkg/ /sbin /usr /home /home/dataiku /runtime-addons; do \
```
Comment (peterableda) [inline rstudio/Dockerfile:139]: dataiku?

File: rstudio/Dockerfile [hunk 5]
```diff
@@ -39,78 +39,124 @@
...
+ if [ -d ${i} ]; then \
+   chown cdsw ${i}; \
+   find ${i} -type d -exec chown cdsw {} +; \
+ fi; \
+ done
 
-# Install additional R dependencies
-COPY r-runtime-dependencies.txt /build/
```
Comment (peterableda) [inline rstudio/Dockerfile:85]: If you don't use this, you should delete the file from the repo too.

File: rstudio/Dockerfile [hunk 6]
```diff
@@ -39,78 +39,124 @@
...
-    apt-get update && \
-    cat /build/r-runtime-dependencies.txt | \
-      sed '/^$/d; /^#/d; s/#.*$//' | \
-      xargs apt-get install -y --no-install-recommends && \
-    rm -rf /var/lib/apt/lists/* && \
-    rm -rf /build
-
-# Change permissions to allow env variables to be pushed to Renviron.site file
-# when the rstudio-cml startup file is run
-RUN chmod -R 666 /etc/R/Renviron.site
```
Comment (peterableda) [inline rstudio/Dockerfile:97]: Why don't you need this anymore?

File: rstudio/Dockerfile (hunk 1 above)
Comment (peterableda) [reply rstudio/Dockerfile:5]: 2024 :)

File: rstudio/Dockerfile (hunk 2 above)
Comment (ogakulov) [reply rstudio/Dockerfile:129]: Removed daiku folder

File: rstudio/Dockerfile (hunk 3 above)
Comment (ogakulov) [reply rstudio/Dockerfile:136]: That was a duplicate

File: rstudio/Dockerfile (hunk 5 above)
Comment (ogakulov) [reply rstudio/Dockerfile:85]: Done

File: rstudio/Dockerfile (hunk 6 above)
Comment (ogakulov) [reply rstudio/Dockerfile:97]: This path doesn't exist in the new build, it seems.

**Pull request closed without merging**